src/js/components/Team/TeamHeader.jsx

```diff
@@ -85,19 +85,17 @@ const OneTeamHeader = styled('div')`
   margin-top: 10px;
 `;
 
-const TeamHeaderCell = styled('div', {
-  shouldForwardProp: (prop) => !['largefont', 'titlecell', 'cellwidth'].includes(prop),
-})(({ largefont, titlecell, cellwidth }) => (`
+const TeamHeaderCell = styled.div`
   align-content: center;
-  ${(titlecell) ? '' : 'border-bottom: 1px solid #ccc;'}
-  ${(largefont) ? 'font-size: 1.1em;' : 'font-size: .8em;'}
-  ${(titlecell) ? '' : 'font-weight: 550;'}
+  border-bottom: ${(props) => (props?.$titleCell ? ';' : '1px solid #ccc;')}
+  font-size: ${(props) => (props?.$largefont ? '1.1em;' : '.8em;')};
```

Comment:
I would prefer to only pass in the property names if they have a value, like this:
${(props) => (props?.$titleCell ? '' : 'border-bottom: 1px solid #ccc;')}
${(props) => (props?.$titleCell ? '' : 'font-weight: 550;')}

One can never be sure that Firefox or Opera will treat properties defined like this as "don't do anything", so I'd rather not send this css to the DOM:
border-bottom: ;
font-weight: ;